src/System.Net.HttpListener/tests/AuthenticationTests.cs

```diff
@@ -47,6 +47,23 @@ public AuthenticationTests()
             }
         }
 
+        [ConditionalTheory(nameof(Helpers) + "." + nameof(Helpers.IsWindowsImplementation))] // Managed implementation connects successfully.
```

Comment:
> do you have more info on why we have that condition and not an ActiveIssue?

So that we can we build the managed implementation on Windows and have msbuild /t:rebuildandtest run all of the tests relevant to the managed implementation rather than relevant to a specific OS.